ui/src/pages/Create.js

```diff
@@ -0,0 +1,645 @@
+  linkedin: {
+    id: 'Create.linkedin',
+    defaultMessage: 'LinkedIn',
+    description: 'The networking site.',
+  },
+  moreDropdown: {
+    id: 'Create.moreDropdown',
+    defaultMessage: 'More&nbsp;&nbsp;&#9207;',
+    description: ('A label on a button to show additional fields that are '
+        + 'hidden by default.'),
+  },
+  orEnterPhotoUrl: {
+    id: 'Create.orEnterPhotoUrl',
+    defaultMessage: 'Or enter photo URL',
+    description: ('A label for a form field where users can submit a photo URL '
+        + '(as opposed to uploading a photo directly)'),
+  },
+  otherWebsite: {
+    id: 'Create.otherWebsite',
+    defaultMessage: 'Other website',
+    description: ('A label for a button for users to add a link to a site '
+        + 'other than a site on a pre-defined list.'),
+  },
+  photo: {
+    id: 'Create.photo',
+    defaultMessage: 'Photo',
+    description: 'A label on a section of a form for providing a photo.',
+  },
+  profilePages: {
+    id: 'Create.profilePages',
+    defaultMessage: 'Profile pages',
+    description: ('A label on a section of a form for adding profile pages '
+        + '(e.g., on Facebook or Twitter).'),
+  },
+  provinceOrState: {
+    id: 'Create.provinceOrState',
+    defaultMessage: 'Province or state',
+    description: 'A label for a form field for a person\'s province or state.',
+  },
+  sex: {
+    id: 'Create.sex',
+    defaultMessage: 'Sex',
+    description: 'A label for a sex (male/female/other) field on a form.',
+  },
+  sexFemale: {
+    id: 'Create.sexFemale',
+    defaultMessage: 'Female',
+    description: 'An option on a form field for a person\'s sex.',
+  },
+  sexMale: {
+    id: 'Create.sexMale',
+    defaultMessage: 'Male',
+    description: 'An option on a form field for a person\'s sex.',
+  },
+  sexOther: {
+    id: 'Create.sexOther',
+    defaultMessage: 'Other',
+    description: 'An option on a form field for a person\'s sex.',
+  },
+  someoneElse: {
+    id: 'Create.someoneElse',
+    defaultMessage: 'Someone else',
+    description: ('A label for a tab for users to submit information about '
+        + 'someone other than themselves.'),
+  },
+  submitRecord: {
+    id: 'Create.submitRecord',
+    defaultMessage: 'Submit record',
+    description: 'A label on a form submission button.',
+  },
+  subscribeToUpdates: {
+    id: 'Create.subscribeToUpdates',
+    defaultMessage: 'Subscribe to updates',
+    description: 'A label on a checkbox to subscribe to updates.',
+  },
+  twitter: {
+    id: 'Create.twitter',
+    defaultMessage: 'Twitter',
+    description: 'The social media site.',
+  },
+  uploadPhoto: {
+    id: 'Create.uploadPhoto',
+    defaultMessage: 'Upload photo',
+    description: 'Label for a button for users who want to upload a photo.',
+  },
+});
+
+/*
+ * The Create page has two tabs: one for adding yourself and one for adding
+ * another person. This enum is used to refer to the two cases.
+ */
+const CREATE_TAB_INDICES = Object.freeze({
```

Comment:
You really don't need CREATE_ for this and below because they are private in this file unless you export them.